.rustfmt.toml

```diff
@@ -0,0 +1,16 @@
+ignore = [
+  "quickwit-proto/src/cluster.rs",
+  "quickwit-proto/src/quickwit.rs",
+  "quickwit-swim",
+]
+
+comment_width = 120
+format_strings = true
+group_imports = "StdExternalCrate"
+imports_granularity = "Module"
+license_template_path = ".license_header.txt"
+normalize_comments = true
+trailing_comma = "Never"
```

Comment:
I prefer the default behaivour, it's easier to extend

Reply:
Agreed. Do folks prefer "always" or "vertical"?
https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=#trailing_comma

Reply:
(the default behavior being "vertical")

Reply:
I assume it's vertical, but I don't really care about the difference between the two

Reply:
vertical is better :)